code/z_adventurezones/space_casino.dm

```diff
@@ -0,0 +1,373 @@
+/*
+    Lythine's space casino prefab
+	Contents:
+	 Item slot machine
+	 Barbuddy
+	 Misc props
+*/
+
+// Item slot machine
+
+/obj/submachine/slot_machine/item
+	name = "Item Slot Machine"
+	desc = "A slot machine that produces items rather than money. Somehow."
+	icon_state = "slotsitem-off"
+	//mats = 50
+	var/uses = 0
+
+	var/list/junktier = list( // junk tier, 68% chance
+		"/obj/item/a_gift/easter",
+		"/obj/item/raw_material/rock",
+		"/obj/item/balloon_animal",
+		"/obj/item/cigpacket",
+		"/obj/item/clothing/shoes/moon",
+		"/obj/item/fish/carp",
+		"/obj/item/instrument/bagpipe",
+		"/obj/item/clothing/under/gimmick/yay",
+		"/obj/item/scrap",
+		"/obj/item/paper_bin",
+		"/obj/item/item_box/stickers",
+		"/obj/item/storage/box/costume/hotdog",
+		"/obj/critter/cockroach",
+		"/obj/item/device/light/flashlight",
+		"/obj/item/kitchen/utensil/knife",
+		"/obj/item/staple_gun",
+		"/obj/item/old_grenade/banana/cheese_sandwich",
+		"/obj/item/old_grenade/banana/banana_corndog",
+		"/obj/item/rubberduck"
+	)
+
+	var/list/usefultier = list( // half decent tier, 30% chance
+		"/obj/item/clothing/gloves/yellow",
+		"/obj/item/bat",
+		"/obj/item/reagent_containers/food/snacks/donkpocket/warm",
+		"/obj/item/device/flash",
+		"/obj/item/clothing/glasses/sunglasses",
+		"/obj/vehicle/skateboard",
+		"/obj/item/storage/firstaid/regular",
+		"/obj/item/clothing/shoes/sandal",
+		"/obj/item/cigpacket/random",
+		"/obj/item/clothing/mask/gas",
+		"/obj/critter/domestic_bee",
+		"/obj/item/storage/firstaid/crit"
+	)
+
+	var/list/raretier = list( // rare tier, 2% chance
```

Comment:
I feel "eh" about the inclusion of the voltron and car, particularly because they're two pretty thematic items and winning them from slots doesn't seem super fitting (though I guess you could argue that finding a voltron in a crate is even less exciting).